doc/doc-site/docs/overview/orders.md

```diff
@@ -215,297 +215,214 @@ Econia does not use a critical height of 1. The actual critical height is define
+That means only prices that are a multiple of \$1000 can be expressed in this configuration.
+Thus a different combination must be chosen, for example, by decreasing decimal tick size by a factor of 1000:
 
-1. 0.0001 `wBTC` decimal lot size (corresponding to 1.779227 `USDC` nominal).
-1. 0.01 `USDC` decimal tick size.
+1. 0.00001 `wBTC` decimal lot size (a value of 0.1779227 `USDC`).
+1. 0.00001 `USDC` decimal tick size.
 
-Here, the resultant quote amount increment corresponds to 1 `USDC` subunit, for a tick size of 1:
+Let's check the minimum and maximum expressible prices with this configuration:
 
 ```python
->>> lot_size_decimal = dec('0.0001')
->>> to_check = (lot_size_decimal * tick_size_decimal
-...             * 10 ** decimals_quote).normalize()
->>> tick_size_integer = int(to_check)
->>> assert to_check >= 1 and to_check == tick_size_integer
->>> tick_size_integer
-1
+>>> lot_size = "0.00001"
+>>> tick_size = "0.00001" # changed by / 1000
+>>> get_min_quote_per_base_nominal(lot_size, tick_size)
+1.0
+>>> get_max_quote_per_base_nominal(lot_size, tick_size)
+4294967295.0
 ```
 
-However, the new decimal order size granularity corresponds to 1.779227 `USDC` nominal, which may not be considered granular enough.
-Hence alternative parametric inputs might entail:
+That's a minimum price (and price granularity) of \$1, as well as a maximum price that's "plenty" high.
+This could work; if a price granularity of one penny is desired, then lower the tick size by a further factor of 100:
 
-1. 0.00005 `wBTC` decimal order size granularity (corresponding to 0.8896135 `USDC` nominal).
-1. 0.02 `USDC` decimal price granularity.
+```python
+>>> lot_size = "0.00001"
+>>> tick_size = "0.0000001" # changed by / 1000 / 100
+>>> get_min_quote_per_base_nominal(lot_size, tick_size)
+0.01
+>>> get_max_quote_per_base_nominal(lot_size, tick_size)
+42949672.95
+```
 
-Here, the total quote amount increment again corresponds to an integer multiple of `USDC` subunits, for a tick size of 1:
+**...but wait a second!** `USDC` *has 6 decimals and above, we've used 7*.
+We can see that this tick size is problematic by using `get_tick_size_integer` from the SDK:
 
-```python
->>> lot_size_decimal = dec('0.00005')
->>> tick_size_decimal = dec('0.02')
->>> to_check = (lot_size_decimal * tick_size_decimal
-...             * 10 ** decimals_quote).normalize()
->>> tick_size_integer = int(to_check)
->>> assert to_check >= 1 and to_check == tick_size_integer
->>> tick_size_integer
-1
 ```
+>>> get_tick_size_integer("0.0000001", 6)
+Traceback (most recent call last):
+  File "<stdin>", line 1, in <module>
+  File "/Users/egd/projects/econia/src/python/sdk/econia_sdk/utils/decimals.py", line 46, in get_tick_size_integer
+    return get_lot_size_integer(smallest_decimal_unit, coin_decimals)
+  File "/Users/egd/projects/econia/src/python/sdk/econia_sdk/utils/decimals.py", line 31, in get_lot_size_integer
```

Comment:
I don't think it's good practice to include personal user names here

And most people stop reading the moment they see a traceback, so I think it's easiest to just show:

```python
>>> get_tick_size_integer("0.0000001", 6)
Traceback (most recent call last):
...
ValueError: Decimal unit too small to represent with 1 subunit
```